src/app/components/cds/inputs/ToggleButtonGroup.js

```diff
@@ -56,15 +69,17 @@ const ToggleButtonGroup = ({
 
 ToggleButtonGroup.defaultProps = {
   className: '',
+  size: 'default',
   helpContent: null,
   label: '',
   variant: 'outlined',
 };
 
 ToggleButtonGroup.propTypes = {
   className: PropTypes.string,
+  size: PropTypes.oneOf(['default', 'small', 'large']),
   helpContent: PropTypes.element,
-  label: PropTypes.string,
+  label: PropTypes.element,
```

Comment:
I think it's better to have label as PropTypes.node as it accounts for strings, number and elements, anything that can be rendered

Reply:
Thanks @amoedoamorim I changed the new proptypes in this PR to be .node instead